zio-json/jvm/src/main/scala/zio/json/JsonDecoderPlatformSpecific.scala

```diff
@@ -30,10 +30,15 @@ trait JsonDecoderPlatformSpecific[A] { self: JsonDecoder[A] =>
   final def decodeJsonStreamInput[R](
     stream: ZStream[R, Throwable, Byte],
     charset: Charset = StandardCharsets.UTF_8
-  ): ZIO[R, Throwable, A] =
-    stream.toInputStream
-      .flatMap(is => ZManaged.fromAutoCloseable(UIO(new java.io.InputStreamReader(is, charset))))
-      .use(readAll)
+  ): ZIO[R with Scope, Throwable, A] =
```

Comment:
This `with Scope` needs to be deleted (!).

Reply:
See #601 for fix.